plugins/main/public/controllers/management/components/management/configuration/integrity-monitoring/integrity-monitoring.js

```diff
@@ -34,15 +34,14 @@ class WzConfigurationIntegrityMonitoring extends Component {
     super(props);
   }
   componentDidMount() {
-    this.props.currentConfig['syscheck-syscheck'].syscheck.disabled = 'no';
+    if (!isString(this.props.currentConfig['syscheck-syscheck']))
+      this.props.currentConfig['syscheck-syscheck'].syscheck.disabled = 'no';
```

Comment:
**nitpick**: the props should not be changed directly. This could cause bugs.

Reply:
Thanks for the suggestion. I fixed it.